### PR TITLE
Fixed some grammaticalities

### PR DESCRIPTION
**Pull request by fritzmark**
While reading this awesome template I stumbled over some wording, typos and grammatical nuisances. Nothing critical. I hope this pull request helps to enhance reading flow.

### CODE REVIEW

File: EN/asciidoc/src/04_solution_strategy.adoc [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
+A short summary and explanation of the fundamental decisions and solution strategies, that shape system architecture. It includes
 
 * technology decisions
 * decisions about the top-level decomposition of the system, e.g. usage of an architectural pattern or design pattern
 * decisions on how to achieve key quality goals
 * relevant organizational decisions, e.g. selecting a development process or delegating certain tasks to third parties.
 
 .Motivation
-These decisions form the cornerstones for your architecture. They are the basis for many other detailed decisions or implementation rules.
+These decisions form the cornerstones for your architecture. They are the base for many other detailed decisions or implementation rules.
```
Comment (gernotstarke) [inline EN/asciidoc/src/04_solution_strategy.adoc:16]: foundation instead of "base"

File: EN/asciidoc/src/04_solution_strategy.adoc [hunk 2]
```diff
@@ -5,21 +5,21 @@
...
 
 .Form
-Keep the explanation of these key decisions short.
+Keep the explanations of key decisions short.
```
Comment (gernotstarke) [inline EN/asciidoc/src/04_solution_strategy.adoc]: such key decisions

**Review by gernotstarke — CHANGES_REQUESTED** [review]
thanx @fritzmark - great you contributed... I was "nitpicking" in your changes...

File: EN/asciidoc/src/04_solution_strategy.adoc (hunk 1 above)
Comment (fritzmark) [reply EN/asciidoc/src/04_solution_strategy.adoc:16]: Right that is much better.

File: EN/asciidoc/src/04_solution_strategy.adoc (hunk 2 above)
Comment (fritzmark) [reply EN/asciidoc/src/04_solution_strategy.adoc]: Yepp makes it more clear and does not sound unneeded.

**Comment by fritzmark** [comment]
Your welcome @gernotstarke.

**Review by gernotstarke — APPROVED** [review]

**Review by gernotstarke — APPROVED** [review]

**Review by gernotstarke — APPROVED** [review]

**Comment by almereyda** [comment]
Please note that such work can also be partially automated with adding a "prose linter" like https://vale.sh/ into the CI pipeline.

I am sparing ourselves an issue, because I don't consider it one, and there are no discussions enabled, why I am leaving this note here.

**Pull request merged**